proxima-overview.md

```diff
@@ -1,36 +1,35 @@
 
 I wish I had that briefcase available in a number of situations, getting a bad grade or running into the inevitable school bully: my curiosity about the relationship between miniaturization and technology remains a fixed point even today.
 
-Then, decades later, the briefcase idea surfaced again while having fun with an iteration of the Oracle’s [Proxima City](https://www.youtube.com/watch?v=aJUSEEagm8Q) model made by the Italian Innovation Team, a whimsical (as it’s been called) construction made with Legos, Arduinos, sensors and Oracle Cloud Infrastructure services.
+Then, decades later, the briefcase idea surfaced again while having fun with an iteration of the Oracle's [Proxima City](https://www.youtube.com/watch?v=aJUSEEagm8Q) model made by the Italian Innovation Team, a whimsical (as it's been called) construction made with Legos, Arduinos, sensors and Oracle Cloud Infrastructure services.
 
 During the 2020 lockdown we had no access to the city model, installed in the Customer Visit Center in Rome. I was struggling to find a way to show a functioning demo either via Zoom or bringing some artifacts into customer premises (no Star Trek transporter available, so far), and urged to prepare something smaller than a city model but powerful enough to show the concept.
 
-“I should definitely do it” was my inner morning echo while sipping my favorite daily americano brew. Changing the model to be portable also meant changing the use cases perimeter: the year 2020 required — for obvious reasons — a rapid transition from Smart City services to an extended and appropriate Safe City scenario.
+"I should definitely do it" was my inner morning echo while sipping my favorite daily americano brew. Changing the model to be portable also meant changing the use cases perimeter: the year 2020 required — for obvious reasons — a rapid transition from Smart City services to an extended and appropriate Safe City scenario.
 
 ![The playground](assets/bread.jpeg)
 
-Requirements were clear: small enough to fit in my backpack, programmable, made with off-the-shelf components, easily connectable to OCI services, and capable to show the end-to-end message flow by means of physical interaction. The stretch goal was to enable teachers, students, developer and tinkerers to improve this ‘portable lab’ and develop new use cases oriented to safety, exploring and expanding the needed back-end cloud services.
+Requirements were clear: small enough to fit in my backpack, programmable, made with off-the-shelf components, easily connectable to OCI services, and capable to show the end-to-end message flow by means of physical interaction. The stretch goal was to enable teachers, students, developer and tinkerers to improve this 'portable lab' and develop new use cases oriented to safety, exploring and expanding the needed back-end cloud services.
 
-## From Briefcase to Cloud, and back
+## From Briefcase to Cloud, and Back
```

Comment:
Per our discussion, let's stick to title casing for all headlines. That does not, however, apply to post/page titles, which I think look better sentence cased. Open to discussion on that.